cli/tests/e2e/snapshots/test_check/test_metavariable_propagation_regex/results.json

```diff
@@ -22,6 +22,19 @@
         "message": "Semgrep found a match foo\n",
         "metadata": {},
         "metavars": {
+          "$1": {
```

Comment:
I thought we were actually considering the numeric metavar differently, and set unify_numeric_metavar to false.